packages/browser/rollup.config.npm.mjs

```diff
@@ -0,0 +1,86 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs'
+import nodeResolve from '@rollup/plugin-node-resolve'
+import replace from '@rollup/plugin-replace'
+import terser from '@rollup/plugin-terser'
+import typescript from '@rollup/plugin-typescript'
+import fs from 'fs'
+
+import createRollupConfig, { sharedOutput as commonSharedOutput } from "../../.rollup/index.mjs"
+
+const packageJson = JSON.parse(fs.readFileSync('./package.json'))
+
+const sharedOutput = {
+  ...commonSharedOutput,
+  strict: false, // 'use strict' in WebKit enables Tail Call Optimization, which breaks stack trace handling
```

Comment:
There is one tail call in out stack and our depth handling does not handle it. With TCO enabled the stack frame for the tail call is removed, which then causes problems with stack processing leading to unexpected results (only in WebKit browsers since that's the only one supporting TCO):

<img width="719" alt="Screenshot 2024-12-27 at 10 14 05" src="https://github.com/user-attachments/assets/4c9173d2-9452-4dea-90af-43a01de2c036" />

Dropping `'use strict'` prevents TCO being enabled and resolves the issue